Android/app/build.gradle

```diff
@@ -8,13 +8,13 @@ plugins {
 
 android {
     namespace "live.dittolive.chat"
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
         applicationId "live.dittolive.chat"
         minSdk 29
-        targetSdk 33
-        versionCode 27
+        targetSdk 34
+        versionCode 32
         versionName "1.0"
```

Comment:
Should the Android `versionName` be set to `5.0.0` to match the version of the iOS app?

Reply:
I personally don't think it needs to match

Reply:
No worries, just thought I'd ask since it looks like we haven't changed it from the default.

Reply:
Updating the version does make sense though. I suggest 1.0.1

Reply:
```suggestion
        versionName "1.0.1"
```